docs/tutorials/Astro/introducing-apollo.md

```diff
@@ -0,0 +1,503 @@
+
+To build and preview your site before deployment, run:
+
+```bash
+cd frontend
+npm run build
+npm run preview
+```
+This serves your built project in a production-like environment, helping you catch any issues before deployment.
+
+With our development environment ready, we need to understand some fundamental concepts before we start building. Both ApostropheCMS and Astro bring their own paradigms to web development, and seeing how they work together will be crucial for building effective websites with the Apollo Theme.
+
+## Core concepts
+
+Current ApostropheCMS users are likely already familiar with its core concepts and components. This section covers those fundamentals and will be especially useful for developers coming from Astro or those new to this stack.
+
+That said, even experienced ApostropheCMS developers will find value here — we’re not just reviewing core concepts, but also how they integrate with Astro.
+
+ApostropheCMS provides a structured approach to managing content and design, and understanding these concepts is key because they require implementation on both the backend (ApostropheCMS) and the frontend (Astro). If you're already comfortable with ApostropheCMS, the transition to using Astro as a frontend isn’t a huge leap—you’ll primarily be shifting how you handle templates.
+
+#### Areas
+[Areas](/guide/areas-and-widgets.html) are editable regions on your pages where content editors can add and arrange content. In the ApostropheCMS backend, you define what types of content (widgets) can be placed in each area and configure their options. In your Astro frontend templates, these areas are rendered using the `AposArea` component, which maintains the in-context editing capabilities of ApostropheCMS while working within Astro's component system.
+
+Here's how an area is configured in your backend:
+
+<AposCodeBlock>
+
+```javascript
+export default {
+  fields: {
+    add: {
+      main: {
+        type: 'area',
+        options: {
+          widgets: {
+            '@apostrophecms/rich-text': {},
+            '@apostrophecms/image': {},
+            'row': {}
+          }
+        }
+      }
+    }
+  }
+};
+```
+<template v-slot:caption>
+  backend/modules/@apostrophecms/home-page/index.js
+</template>
+</AposCodeBlock>
+
+This area would allow for content managers to add any number of three different widget types - rich-text, image, and row. You can [read about](/guide/areas-and-widgets.html) additional configuration options, like limiting the number of widgets that can be added, and passing options to the individual widgets.
+
+::: info
+Note two things:
+  - the widgets are passed into the area options without including the `-widget` suffix
+  - widgets that come from the ApostropheCMS core are prefixed with `@apostrophecms/` - this is also true for any other widgets that come from packages in the `node_modules`
```

Comment:
"This is also true for any other widgets that come from official ApostropheCMS npm pages" would be more accurate.

Reply:
Corrected.